<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExitDialog</class>
 <widget class="QDialog" name="ExitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="message_screen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Shutdown ATM?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_yes">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_no">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
